gradle.properties

```diff
@@ -1 +1 @@
-gradleRioVersion: 2019.4.1
+gradleRioVersion: 2020.0.0-alpha-2

```

Comment:
Are the line endings ok in this file?

Reply:
Yes, this is fine, it really weird why its happening but it works just fine.